routers/web/org/home.go

```diff
@@ -155,5 +157,31 @@ func Home(ctx *context.Context) {
 
 	ctx.Data["ShowMemberAndTeamTab"] = ctx.Org.IsMember || len(members) > 0
 
+	profileGitRepo, profileReadmeBlob, profileClose := shared_user.FindUserProfileReadme(ctx)
+	defer profileClose()
+	calcOrgProfileReadme(ctx, profileGitRepo, profileReadmeBlob)
+
 	ctx.HTML(http.StatusOK, tplOrgHome)
 }
+
+func calcOrgProfileReadme(ctx *context.Context, profileGitRepo *git.Repository, profileReadme *git.Blob) {
+	if profileGitRepo == nil || profileReadme == nil {
+		ctx.Data["HasProfileReadme"] = false
```

Comment:
This variable is not be necessary. Just use `{{if .ProfileReadme}}` in the template.